### PR TITLE
Backports for 0.12.0 (updated to e289807)

### PR DESCRIPTION
**Pull request by luke-jr**
Somehow these got missed

### CODE REVIEW

**Comment by MarcoFalke** [comment]
At least two of them got missed on purpose.

File: doc/release-notes.md [hunk 1]
```diff
@@ -350,6 +350,20 @@ caching. A sample config for apache2 could look like:
 
     </VirtualHost>
 
+Asm script outputs now contain OP_CHECKLOCKTIMEVERIFY in place of OP_NOP2
```
Comment (laanwj) [inline doc/release-notes.md]: I'd suggest combining this with the other asm change related to signature flags, which is currently under "RPC: Low-level API changes".

**Comment by laanwj** [comment]
utACK now that it contains no translation string changes anymore

File: doc/release-notes.md (hunk 1 above)
Comment (luke-jr) [inline doc/release-notes.md]: Done

**Comment by MarcoFalke** [comment]
utACK e20704b

Missing: #7327

**Pull request merged**